src/lib/config/defaultConfig.js

```diff
@@ -13,7 +13,8 @@ export const APP_CONFIG = {
   SEARCH_READY_DELAY: 2000,
   INVENIO_UI_URL: process.env.REACT_APP_INVENIO_UI_URL,
   LOGO_SRC: process.env.PUBLIC_URL + '/logo-invenio-ils.svg',
-  MAX_RESULTS_WINDOW: 10000,
+  MAX_RESULTS_WINDOW: 5000,
```

Comment:
I would change it to the following:
```suggestion
  SEARCH_MAX_RESULTS: 10000,  // should match search `max_results_window` in the backend
  VOCABULARIES_MAX_RESULTS: 500,
  EXPORT_MAX_RESULTS: 5000,
```
This is because the backend allows 10K, and that number is used to calculate the pagination. If we keep it aligned with the backend, we can actually jump to that page with the pagination.

Then, for vocab, perfect.
For the export, since we need to export all of them at one, I would indeed introduce a new var but not change the default one.

WDYT?